nuxeo-services/nuxeo-platform-convert/src/main/java/org/nuxeo/ecm/platform/convert/plugins/IWork2PDFConverter.java

```diff
@@ -58,8 +59,14 @@ public BlobHolder convert(BlobHolder blobHolder, Map<String, Serializable> param
             if (mimeType == null || !IWORK_MIME_TYPES.contains(mimeType)) {
                 throw new ConversionException("not an iWork file");
             }
+            // check if the stream represents a valid zip
+            blobStream = blob.getStream();
+            if (!ZipUtils.isValid(blobStream)) {
+                throw new ConversionException("not a valid iWork file");
+            }
```

Comment:
Please use a try-with-resources pattern:

    // check if the stream represents a valid zip
    try (InputStream blobStream = blob.getStream()) {
        if (!ZipUtils.isValid(blobStream)) {
            throw new ConversionException("not a valid iWork file");
        }
    }